src/hip_ra_x/hip_ra_x.py

```diff
@@ -777,10 +813,17 @@ def render_scientific(p: floatParameter | OutputParameter) -> str:
                 f.write(f'                               ***HIP CASE REPORT***{nl}')
                 f.write(f'                               *********************{nl}')
                 f.write(nl)
-                f.write(f'                           ***SUMMARY OF RESULTS***{nl}')
-                f.write(nl)
+                f.write(f'      ***SUMMARY OF INPUTS***{nl}')
 
-                for k, v in case_data['SUMMARY OF RESULTS'].items():
+                for k, v in case_data_inputs['SUMMARY OF INPUTS'].items():
+                    # align space between value and units to same column
+                    kv_spaces = max(1, (24 - (len(v.split(' ')[0]) + len(k)))) * ' '
+
+                    f.write(f'      {k}:{kv_spaces}{v}{nl}')
+
+                f.write(nl)
+                f.write(f'      **SUMMARY OF RESULTS***{nl}')
```

Comment:
Heading asterisks are off, was 3 on each side, here it's 2 on the left and 4 on the right